pyproject.toml

```diff
@@ -9,15 +9,14 @@ name = "statannotations"
 description = "add statistical significance or custom annotations on seaborn plots. Based on statannot 0.2.3"
 readme = "README.md"
 license = {text = "MIT License"}
-requires-python = ">=3.7"
+requires-python = ">=3.8"
```

Comment:
Ah forgot about this one... Do you mind if I cherry-pick this one independently?

Reply:
Yes, sure!
If you are planning a patch release, I realized the README also needs minor corrections: mention that the last version is 0.6, should be 0.7.